src/sentry/static/sentry/app/components/group/releaseStats.jsx

```diff
@@ -67,7 +67,8 @@ const GroupReleaseStats = createReactClass({
       this.state.loading !== nextState.loading ||
       this.state.error !== nextState.error ||
       this.state.environment !== nextState.environment ||
-      this.props.group.id !== nextProps.group.id
+      this.props.group.id !== nextProps.group.i ||
```

Comment:
group.i < id

is this the entire state and props? what is it here that we don't want to update one? could we delete the method or use a deep compare util?

Reply:
Agh, thanks

Reply:
I'm not sure why we need this. It seems that we're just comparing all the props and state that would reasonably change anyway.